CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # ChangeLog
 
+## 3.1.0 - 2023-06-29
+
+Changes:
+- add new BBB 2.6 API parameters
+  - notifyRecordingIsOn
+  - prsentationUploadExternalUrl
+  - prsentationUploadExternalDescription
+  - recordFullDurationMedia (v2.6.9)
+  - disabledFeaturesExclude(2.6.9)
+  - userdata-bbb_hide_presentation_on_join
+- move configuration of checksum hash function from `ClusterGroups` to `Clusters` 
```

Comment:
Are ClusterGroup settings somehow migrated? Changelog should have a more detailed note about it.

(I would prefer a migration a settings from *ClusterGroup* to group, although it may be not deterministic which value is chosen if a *Cluster* is part of multiple *ClusterGroup*s)

Reply:
@liske migration is already included [here](https://github.com/DE-IBH/b3lb/blob/114-support-for-bbb-v26/b3lb/rest/migrations/0013_update_sha_and_parameters.py#L5-L9)